examples/pip/extras/BUILD

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load("@examples_extras//:requirements.bzl", "requirement")
-load("//python:defs.bzl", "py_test")
+load("@extras_deps//:requirements.bzl", "requirement")
+load("@rules_python//python:defs.bzl", "py_test")
 
 package(default_visibility = ["//visibility:public"])
```

Comment:
Not directly related to your change, but this looks to be unnecessary, and I'm not sure we want to leave it here in case it leads to newbies copy-pasting it around their BUILD files (I did this in my first Bazel migration).

Reply:
good callout, agreed we should train users to default to `["//:__subpackages__"]` or more specific where possible.